pages/article/index.page.tsx

```diff
@@ -3,11 +3,10 @@ import type { GetStaticProps, InferGetStaticPropsType } from 'next';
 import { connectMongo } from 'pages/api/middlewares/connectMongo';
 import ArticleModel from 'pages/api/models/articleModel';
 
-import { Article, SideTab } from 'src/components';
-import { Layout, Grid } from 'src/components/ui';
+import { ArticleList, Hero } from 'src/components';
 import { getPlaiceholder } from 'plaiceholder';
 
-type ArticleT = {
+export type ArticleT = {
```

Comment:
```suggestion
export type TArticle = {
```

그런데 ArticleModel 이라는게 있고, ArticleT 라는게 있네요. 둘다 동일해야할 것 같은데 뭐가 다르나요? 네이밍을 다시 해보는게 어떨까요

Reply:
네 네이밍을 한번 다시 해봐야할 것 같습니다. AritcleModel을 제가 작성해놓고 까먹고 있었네요.. 다른 커멘트에서 말씀하신 것처럼 차라리 공통된 타입들은 모아놓은 파일을 따로 만들어서 관리해보도록 하겠습니다.